### DISCUSSION TITLE
2022/06/14/%E3%80%8E%E9%9A%8F%E7%AC%94%E3%80%8F618%E6%A1%8C%E9%9D%A2%E6%94%B9%E9%80%A0%E8%AE%A1%E5%88%92/

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#17] [General]
# 2022/06/14/%E3%80%8E%E9%9A%8F%E7%AC%94%E3%80%8F618%E6%A1%8C%E9%9D%A2%E6%94%B9%E9%80%A0%E8%AE%A1%E5%88%92/

前情提要 鄙人自开学以来桌面一直是没有放显示器的，毕竟从小到大一直用小屏的笔记本电脑打游戏，上了大学也没有什么感觉。我高考后买的一台屏幕 13 寸的 ROG 幻13，可以 360° 旋转，还支持触控，加上一支微软的 Surface Pen，上课记笔记就变得十分方便，也就一直用到了现在，直到—— 直到

https://nx-official.github.io/2022/06/14/%E3%80%8E%E9%9A%8F%E7%AC%94%E3%80%8F618%E6%A1%8C%E9%9D%A2%E6%94%B9%E9%80%A0%E8%AE%A1%E5%88%92/?highlight=618

**Comment by ZhangStudyLife** [comment]
救命，本来就是给自己网站配个域名，后面好多人反应加载慢，我又换到vercel，然后看了看您的说起来vercel，自从看了您被坑了1000块，我就想给自己加个图床(现在自己还是用在本地，用Typora快速导入图片)，一篇一篇博客看下来，看到这篇我只能说您真的是我**启蒙老师**啊啊,桌搭确实很重要，心情很愉悦（我现在桌子乱成一团
